jupyterlab/commands.py

```diff
@@ -315,6 +317,8 @@ def __init__(self, app_dir, logger=None, kill_event=None):
         self.logger = logger or logging.getLogger('jupyterlab')
         self.info = self._get_app_info()
         self.kill_event = kill_event or Event()
+        # TODO: Make this configurable
+        self.registry = 'https://registry.npmjs.org'
```

Comment:
This is in the `lerna.json` file in the repo root.

Reply:
Actually that wouldn't help is because it isn't shipped. Will think about this more...

Reply:
Yeah I think leaving it as is and making it part of the extension application config sounds good